### DISCUSSION TITLE
Excel ProForma

### DISCUSSION THREAD

**Discussion by mrasoana** [NREL/REopt-API-Analysis#134] [V3 Transition]
Hi, the link to download the spreadsheet proforma does not show up anymore. I tried to get one from previous version projects, the only option is to download the PDF results

**Comment by adfarth** [comment]
The REopt team is working on providing the proforma download! The transition of REopt to version 3 made significant breaking changes in the creation of the proforma, which the team has been working to address. We will make an announcement on this page when the proforma is available.
